hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/AbstractRecordStore.java

```diff
@@ -268,11 +284,12 @@ protected void updateStatsOnGet(long now) {
     }
 
     protected void updateStatsOnRemove(long hits) {
```

Comment:
do we need this method any more?

Reply:
I left it there as a placeholder, similarly to `updateStatsOnPut/Get` methods, so that when we need to update stats on remove we won't have to look all over the codebase (I see 6 usages).